src/Restore.php

```diff
@@ -386,7 +386,7 @@ function ($bucket) {
             $headerFile->writeRow($tableInfo['columns']);
 
             $isTyped = $tableInfo['isTyped'] ?? false;
-            if ($this->projectHasFeature('tables-definition') && $isTyped) {
```

Comment:
Tady by spíš mělo být jen pokud má projekt `native-types` ne? Takhle budeš zkoušet restore netypované tabulky i v projektu kde to není zapnuté.

Reply:
pokud děláš restore tabulky co má typy tak chceš restornůt tabulku co má typy ne sní udělat varchar tabulku.

Reply:
No ale ten restore můžu dělat do projektu, který native-types nemá. Takže bys měl validovat jak feature, tak isTyped.

Reply:
feature native-types je pro runner ne pro connection. Tabulku s typama možeš mět kdekoliv.

Reply:
Aha pravda. Tak uz mlcim 🔫

Reply:
fixed